### PR TITLE
Suggest customers to upvote existing issues rather than file a new one

### PR DESCRIPTION
**Pull request by mkArtakMSFT**

### CODE REVIEW

**Comment by mkArtakMSFT** [comment]
@danroth27 I remember you asked about something similar in the past. Let me know if you want me to tweak the wording.

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -6,7 +6,11 @@ One of the easiest ways to contribute is to participate in discussions on GitHub
 Start a discussion on the [repository issue tracker](https://github.com/dotnet/aspnetcore/issues).
 
 ## Bugs and feature requests?
-For non-security related bugs, log a new issue in the appropriate GitHub repository. Here are some of the most common repositories:
+**IMPORTANT: If you want to report a security related issue, please see the `Reporting security issues and bugs` section below.**
+
+Before reporting a non-security related bug or filing a new issue try to find an existing issue if such exists. If found, upvote (👍) it and add a comment about your scenarioto to that issue. Our team pays attention to reactions a lot and having an issue with many upvotes or other positive reactions will help us prioritize it to be worked on sooner rather than later.
```
Comment (BrennanConroy) [inline CONTRIBUTING.md:11]:
```suggestion
Before reporting a non-security related bug or filing a new issue try to find an existing issue if such exists. If found, upvote (👍) it and add a comment about your scenario to that issue. Our team pays attention to reactions a lot and having an issue with many upvotes or other positive reactions will help us prioritize it to be worked on sooner rather than later.
```

Comment (BrennanConroy) [reply CONTRIBUTING.md:11]: Maybe it's just me, but I'm not a fan of people commenting the same thing over and over on an issue, and I definitely don't want to encourage people to do that.

Maybe you can reword this to say something like "add a comment about any unique scenarios you have".

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -6,7 +6,11 @@ One of the easiest ways to contribute is to participate in discussions on GitHub
 Start a discussion on the [repository issue tracker](https://github.com/dotnet/aspnetcore/issues).
 
 ## Bugs and feature requests?
-For non-security related bugs, log a new issue in the appropriate GitHub repository. Here are some of the most common repositories:
+**IMPORTANT: If you want to report a security related issue, please see the `Reporting security issues and bugs` section below.**
+
+Before reporting a non-security related bug or filing a new issue try to find an existing issue if such exists. If found, upvote (👍) it and add a comment about your scenario to that issue. Our team pays attention to reactions a lot and having an issue with many upvotes or other positive reactions will help us prioritize it to be worked on sooner rather than later.
```
Comment (mkArtakMSFT) [inline CONTRIBUTING.md:11]:
```suggestion
Before reporting a non-security related bug or filing a new issue try to find an existing issue if such exists. If found, upvote (👍) it and add a comment about your unique scenario to that issue. Our team pays attention to reactions a lot and having an issue with many upvotes or other positive reactions will help us prioritize it to be worked on sooner rather than later.
```

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -8,7 +8,7 @@ Start a discussion on the [repository issue tracker](https://github.com/dotnet/a
 ## Bugs and feature requests?
 **IMPORTANT: If you want to report a security related issue, please see the `Reporting security issues and bugs` section below.**
 
-Before reporting a non-security related bug or filing a new issue try to find an existing issue if such exists. If found, upvote (👍) it and add a comment about your scenario to that issue. Our team pays attention to reactions a lot and having an issue with many upvotes or other positive reactions will help us prioritize it to be worked on sooner rather than later.
+Before reporting a non-security related bug or filing a new issue try to find an existing issue if such exists. If found, upvote (👍) it and add a comment about your unique scenario to that issue. Our team pays attention to reactions a lot and having an issue with many upvotes or other positive reactions will help us prioritize it to be worked on sooner rather than later.
```
Comment (danroth27) [inline CONTRIBUTING.md:11]:
```suggestion
Before reporting a new issue, try to find an existing issue if one already exists. If it already exists, upvote (👍) it. Also consider adding a comment with your unique scenarios and requirements related to that issue.  Upvotes and clear details on the issue's impact helps us prioritize the most important issues to be worked on sooner rather than later.
```

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -6,7 +6,11 @@ One of the easiest ways to contribute is to participate in discussions on GitHub
 Start a discussion on the [repository issue tracker](https://github.com/dotnet/aspnetcore/issues).
 
 ## Bugs and feature requests?
-For non-security related bugs, log a new issue in the appropriate GitHub repository. Here are some of the most common repositories:
+**IMPORTANT: If you want to report a security related issue, please see the `Reporting security issues and bugs` section below.**
```
Comment (Pilchie) [inline CONTRIBUTING.md:9]: Consider putting a ❗ icon before the "IMPORTANT".

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -6,7 +6,11 @@ One of the easiest ways to contribute is to participate in discussions on GitHub
...
+
+Before reporting a new issue, try to find an existing issue if one already exists. If it already exists, upvote (👍) it. Also consider adding a comment with your unique scenarios and requirements related to that issue.  Upvotes and clear details on the issue's impact help us prioritize the most important issues to be worked on sooner rather than later.
```
Comment (Pilchie) [inline CONTRIBUTING.md:11]: I
```suggestion
Before reporting a new issue, try to find an existing issue if one already exists. If it already exists, upvote (👍) it. Also consider adding a comment with your unique scenarios and requirements related to that issue.  Upvotes and clear details on the issue's impact help us prioritize the most important issues to be worked on sooner rather than later. If you can't find one, that's okay, we'd rather a duplicate report than none.
```

Comment (Pilchie) [reply CONTRIBUTING.md:11]: We should be careful not to scare people away by having language that's too strong.

**Review by Pilchie — APPROVED** [review]

File: CONTRIBUTING.md [hunk 6]
```diff
@@ -6,7 +6,11 @@ One of the easiest ways to contribute is to participate in discussions on GitHub
 Start a discussion on the [repository issue tracker](https://github.com/dotnet/aspnetcore/issues).
 
 ## Bugs and feature requests?
-For non-security related bugs, log a new issue in the appropriate GitHub repository. Here are some of the most common repositories:
+❗ **IMPORTANT: If you want to report a security related issue, please see the `Reporting security issues and bugs` section below.**
+
+Before reporting a new issue, try to find an existing issue if one already exists. If it already exists, upvote (👍) it. Also consider adding a comment with your unique scenarios and requirements related to that issue.  Upvotes and clear details on the issue's impact help us prioritize the most important issues to be worked on sooner rather than later. If you can't find one, that's okay, we'd rather a duplicate report than none.
```
Comment (mkArtakMSFT) [inline CONTRIBUTING.md:11]:
```suggestion
Before reporting a new issue, try to find an existing issue if one already exists. If it already exists, upvote (👍) it. Also consider adding a comment with your unique scenarios and requirements related to that issue.  Upvotes and clear details on the issue's impact help us prioritize the most important issues to be worked on sooner rather than later. If you can't find one, that's okay, we'd rather have a duplicate report than none.
```

**Review by danroth27 — APPROVED** [review]
